.github/renovate.json5

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Let's rename this to `renovate-aws-marketplace.json5` and import in the main `renovate.json5`.